client/src/rpb/TablePicker.js

```diff
@@ -10,7 +10,8 @@ import { AlertBanner, GlossaryIcon } from "../components";
 import {
   categories,
   concepts_by_category,
-  concept_filter,
+  concept_white_filter,
+  concept_filter_by_type,
```

Comment:
Trying to figure out what these are by prop name alone, and having a pretty hard time. Better names you can give these?

Reply:
`concept_whitelist_filter`
`concept_filter_by_categories` ?

Reply:
I can leave a comment in the code for `concept_filter_by_type` if it's not clear